website/www/site/content/en/get-started/downloads.md

```diff
@@ -96,14 +96,22 @@ versions denoted `0.x.y`.
 
 ## Releases
 
-### 2.50.0 (2023-08-30)
+### 2.51.0 (2023-10-03)
 Official [source code download](https://downloads.apache.org/beam/2.50.0/apache-beam-2.50.0-source-release.zip).
```

Comment:
These links needs to be updated for 2.51.0.

Reply:
Done